group-coordinator/src/main/java/org/apache/kafka/coordinator/group/OffsetConfig.java

```diff
@@ -88,10 +125,10 @@ public OffsetConfig(int maxMetadataSize,
     }
 
     public OffsetConfig() {
```

Comment:
it seems this constructor is unused. Could we remove it?

Reply:
deleted